packages/js/src/ui/components/Notification/NotificationList.tsx

```diff
@@ -0,0 +1,111 @@
+import { For, ParentProps, Show } from 'solid-js';
```

Comment:
✏ **suggestion:** please split this file to the separate components as it's hard to read